
From: Neil Brown <neilb@cse.unsw.edu.au>

While %Lu works with gcc, %llu is the correct usage.

Signed-off-by: Neil Brown <neilb@cse.unsw.edu.au>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/md/md.c        |    2 +-
 25-akpm/drivers/md/raid5.c     |    2 +-
 25-akpm/drivers/md/raid6main.c |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff -puN drivers/md/md.c~md-convert-lu-to-llu-in-printk drivers/md/md.c
--- 25/drivers/md/md.c~md-convert-lu-to-llu-in-printk	2004-10-10 23:48:54.322104704 -0700
+++ 25-akpm/drivers/md/md.c	2004-10-10 23:48:54.333103032 -0700
@@ -3467,7 +3467,7 @@ static void md_do_sync(mddev_t *mddev)
 	 * Tune reconstruction:
 	 */
 	window = 32*(PAGE_SIZE/512);
-	printk(KERN_INFO "md: using %dk window, over a total of %Lu blocks.\n",
+	printk(KERN_INFO "md: using %dk window, over a total of %llu blocks.\n",
 		window/2,(unsigned long long) max_sectors/2);
 
 	atomic_set(&mddev->recovery_active, 0);
diff -puN drivers/md/raid5.c~md-convert-lu-to-llu-in-printk drivers/md/raid5.c
--- 25/drivers/md/raid5.c~md-convert-lu-to-llu-in-printk	2004-10-10 23:48:54.324104400 -0700
+++ 25-akpm/drivers/md/raid5.c	2004-10-10 23:48:54.335102728 -0700
@@ -1406,7 +1406,7 @@ static int make_request (request_queue_t
 		new_sector = raid5_compute_sector(logical_sector,
 						  raid_disks, data_disks, &dd_idx, &pd_idx, conf);
 
-		PRINTK("raid5: make_request, sector %Lu logical %Lu\n",
+		PRINTK("raid5: make_request, sector %llu logical %llu\n",
 			(unsigned long long)new_sector, 
 			(unsigned long long)logical_sector);
 
diff -puN drivers/md/raid6main.c~md-convert-lu-to-llu-in-printk drivers/md/raid6main.c
--- 25/drivers/md/raid6main.c~md-convert-lu-to-llu-in-printk	2004-10-10 23:48:54.325104248 -0700
+++ 25-akpm/drivers/md/raid6main.c	2004-10-10 23:48:54.337102424 -0700
@@ -1569,7 +1569,7 @@ static int make_request (request_queue_t
 		new_sector = raid6_compute_sector(logical_sector,
 						  raid_disks, data_disks, &dd_idx, &pd_idx, conf);
 
-		PRINTK("raid6: make_request, sector %Lu logical %Lu\n",
+		PRINTK("raid6: make_request, sector %llu logical %llu\n",
 		       (unsigned long long)new_sector,
 		       (unsigned long long)logical_sector);
 
_
